heron/packing/tests/java/com/twitter/heron/packing/roundrobin/ResourceCompliantRRPackingTest.java

```diff
@@ -0,0 +1,570 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.packing.roundrobin;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.twitter.heron.api.generated.TopologyAPI;
+import com.twitter.heron.spi.common.ClusterDefaults;
+import com.twitter.heron.spi.common.Config;
+import com.twitter.heron.spi.common.Constants;
+import com.twitter.heron.spi.common.Context;
+import com.twitter.heron.spi.common.Keys;
+import com.twitter.heron.spi.packing.PackingPlan;
+import com.twitter.heron.spi.utils.TopologyTests;
+import com.twitter.heron.spi.utils.TopologyUtils;
+
+
+public class ResourceCompliantRRPackingTest {
+
+  private static final String BOLT_NAME = "bolt";
+  private static final String SPOUT_NAME = "spout";
+  private static final double DELTA = 0.1;
+
+  private long instanceRamDefault;
+  private double instanceCpuDefault;
+  private long instanceDiskDefault;
+
+
+  private int countCompoment(String component, Map<String, PackingPlan.InstancePlan> instances) {
+    int count = 0;
+    for (PackingPlan.InstancePlan pair : instances.values()) {
+      if (component.equals(RoundRobinPacking.getComponentName(pair.id))) {
+        count++;
+      }
+    }
+    return count;
+  }
+
+  protected TopologyAPI.Topology getTopology(
```

Comment:
@billonahill This is the same as in current round robin packing test. Should I keep it as is or change it in both places?